source/html/renderer/src/host-config.ts

```diff
@@ -17,46 +17,56 @@ function parseSpacingDefinition(obj: any): ISpacingDefinition {
     } : null;
 }
 
-export interface IPaddingDefinition {
-    top: Enums.Padding,
-    right: Enums.Padding,
-    bottom: Enums.Padding,
-    left: Enums.Padding
+export class PaddingDefinition {
+    top: Enums.Padding;
+    right: Enums.Padding;
+    bottom: Enums.Padding;
+    left: Enums.Padding;
+
+    toJSON() {
+        return {
+            top: Enums.Padding[this.top],
+            right: Enums.Padding[this.right],
+            bottom: Enums.Padding[this.bottom],
+            left: Enums.Padding[this.left],
+        }
+    }
 }
 
-function parsePaddingDefinition(obj: any): IPaddingDefinition {
-    return obj ? {
+function parsePaddingDefinition(obj: any): PaddingDefinition {
+    return obj ? Object.assign(new PaddingDefinition(), {
         top: Utils.parseHostConfigEnum(Enums.Padding, obj["top"], Enums.Padding.None),
         right: Utils.parseHostConfigEnum(Enums.Padding, obj["right"], Enums.Padding.None),
         bottom: Utils.parseHostConfigEnum(Enums.Padding, obj["bottom"], Enums.Padding.None),
         left: Utils.parseHostConfigEnum(Enums.Padding, obj["left"], Enums.Padding.None),
-    } : null;
+    }
+    ) : null;
 }
 
-export interface ITextColorDefinition {
-    normal: string,
-    subtle: string
+export class TextColorDefinition {
+    normal: string;
+    subtle: string;
 }
 
-function parseTextColorDefinition(obj: any): ITextColorDefinition {
+function parseTextColorDefinition(obj: any): TextColorDefinition {
     return obj ? {
         normal: obj["normal"],
         subtle: obj["subtle"]
     } : null;
 }
 
-export interface IContainerStyleDefinition {
-    backgroundColor?: string,
+export class ContainerStyleDefinition {
+    backgroundColor?: string;
     fontColors: {
-        default: ITextColorDefinition,
-        accent: ITextColorDefinition,
-        good: ITextColorDefinition,
-        warning: ITextColorDefinition,
-        attention: ITextColorDefinition
-    },
+        default: TextColorDefinition,
+        accent: TextColorDefinition,
+        good: TextColorDefinition,
+        warning: TextColorDefinition,
+        attention: TextColorDefinition
+    };
 }
 
-function parseColorPaletteDefinition(obj: any): IContainerStyleDefinition {
+function parseColorPaletteDefinition(obj: any): ContainerStyleDefinition {
```

Comment:
Since ContainerStyleDefinition is now a class, this method should be moved to the class and should be renamed into simply "parse"